app/views/events/_form.html.erb

```diff
@@ -1,5 +1,19 @@
 <%= form_with model: @event, local: true do |form| %>
 
+ <% if @event.errors.any? %>
+    <div id="error_explanation">
+      <h3>
+        Sorry, <%= pluralize(@event.errors.count, "error") %> prohibited
+        this event from being saved:
+      </h3>
+      <ul>
+        <% @event.errors.full_messages.each do |msg| %>
+          <li><%= msg %></li>
+        <% end %>
+      </ul>
+        <h3>Please try again...</h3>
```

Comment:
Wir sollten nicht zu viele headline tags benutzen. Ist ja nur eine Fehlermeldung. Aber ich schlage vor, dass wir solche Stylingsachen bei Gelegenheit in einem Abwasch machen